api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /topics:
+    get:
+      description: Lists all known `Topics` of this Nakadi cluster
+      tags:
+        - monitoring
+        - management-api
+      parameters:
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+      responses:
+        '200':
+          description: The request was successful. Returns a list `Topic`s in the response body.
+          schema:
+            type: array
+            description: An array of topics
+            items:
+              $ref: '#/definitions/Topic'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/topics/{topic}/events':
+    get:
+      tags:
+        - topic-api
+        - draft
+      description: |
+        Starts a stream delivery for the specified partitions of the given topic.
+
+        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
+        `EventStreamBatch` constains a chunk of Events and a `Cursor` pointing to the end of the chunk (i.e.
+        last delivered Event).
+        If streaming for several distinct partitions, each one is a independent `EventStreamBatch`.
+
+        The initialization of a stream can be parameterized in terms of size of each chunk, timeout for
+        flushing each chunk, total amount of delivered Events and total time for the duration of the stream.
+
+        Nakadi will keep a streaming connection open even if there are no events to be delivered. In this
+        case the timeout for the flushing of each chunk will still apply and the `EventStreamBatche`s will
+        contain only the Cursor pointing to the same offset. This can be treated as a keep-alive control for
+        some load balancers.
+
+        The tracking of the current position in the partitions and of which partitions is being read is in
+        the responsibility of the client.
+        No commits are needed.
+      parameters:
+        - name: topic
+          in: path
+          description: Topic where to get events from
+          type: string
+          required: true
+        - name: x-nakadi-cursors
+          in: header
+          description: Array of cursors indicating the partitions to read from and the starting point on
+            each. Assumes the offset on each cursor is not inclusive (i.e., first delivered Event is the
+            **first one after** the one pointed to in the cursor).
```

Comment:
I would prefer having the cursors as part of the url, i.e. passed via url query string.